app/views/posts/index.html.erb

```diff
@@ -1,5 +1,5 @@
 <% @posts.each do |post| %>
-  <p><%= post.message %></p>
+  <p><%= post.message %> updated on <%=post.updated_at.strftime("%d-%m-%Y")%></p>
```

Comment:
Are the posts from the same day are still in reverse chronological order even though the code doesn't make a note of the time it was posted?